go/pkg/client/cas.go

```diff
@@ -668,7 +668,7 @@ func copyFile(execRoot, from, to string) error {
 	return err
 }
 
-func (c *Client) downloadFiles(ctx context.Context, execRoot string, outputs map[digest.Digest]*tree.Output) error {
+func (c *Client) DownloadFiles(ctx context.Context, execRoot string, outputs map[digest.Digest]*tree.Output) error {
```

Comment:
Can you add a comment for this function? golint would've caught this. I'll file a bug to set it up on this repo.

Reply:
done.